src/common/config_opts.h

```diff
@@ -1390,6 +1390,7 @@ OPTION(rbd_mirroring_replay_delay, OPT_INT, 0) // time-delay in seconds for rbd-
  * rbd_create3()/RBD::create3() and rbd_clone2/RBD::clone2() are only
  * affected by rbd_default_order.
  */
+OPTION(rbd_default_pool, OPT_STR, "rbd") // default pool for storing images
```

Comment:
Placing this option here might be confusing. The comment above says "rbd_create()/RBD::create() are affected by all of these options", while actually they are not affected by "rbd_default_pool".